lib/porky_lib/file_service_helper.rb

```diff
@@ -4,11 +4,11 @@
 
 module PorkyLib::FileServiceHelper
   def file_size_invalid?(file)
-    file.bytesize > max_size || (File.file?(file) && File.size(file) > max_size)
+    (file.is_a?(String) && file.bytesize > max_size) || (!file.is_a?(String) && File.size(file) > max_size)
```

Comment:
Looks good but maybe a little bit difficult to read.
2 suggestions:
- So it looks like this method accepts a file or a string (file content) right? Maybe it's counterintuitive to think `file.is_a?(String)` could be true if it's a file. I'd rename it to `file_or_content` or that suggests the accepted formats.
- Maybe add a method to isolate the `is_a_file?` functionality to encapsulate `is_a?(String)` logic. I'd do that to make it more readable. Also because that was the cause of the issue (`File.file?`), that way we could recover easily if it fails again.

Reply:
Yeah you're right, I'll do the those changes